test/features/login_redirect.feature

```diff
@@ -22,23 +22,15 @@ Feature: Login Redirection
         When I log in directly
         Then I should see "Change details"
 
-    @dataset_setup
-    Scenario: As a Sysadmin I set the visibility of a public record to private for the following scenarios
```

Comment:
very good pickup on not changing data mid way through (unless we create a new one during test background)